jax/_src/array.py

```diff
@@ -680,54 +680,85 @@ def make_array_from_callback(
 def make_array_from_single_device_arrays(
     shape: Shape, sharding: Sharding, arrays: Sequence[basearray.Array]
 ) -> ArrayImpl:
-  r"""Returns a ``jax.Array`` from a sequence of ``jax.Array``\s on a single device.
+  r"""Returns a ``jax.Array`` from a sequence of ``jax.Array``\s each on an individual device.
 
-  You can use this function if you have already ``jax.device_put`` the value on
-  a single device and want to create a global Array. The smaller ``jax.Array``\s should be
-  addressable and belong to the current process.
+  To use this function, every device in the global mesh must have an addressable array that
+  is already on on-device, typically either via ``jax.device_put`` or generated directly onto the device.
+
+  The ``jax.Array``\s forming the input argument ``arrays`` must be local to the calling
```

Comment:
Remove this sentence. I think it's confusing and using "local" is not the recommended terminology for single controller runtimes.

Reply:
" must be addressable to the calling"